src/core/create_compilers/RollupCompiler.ts

```diff
@@ -378,7 +378,7 @@ export default class RollupCompiler {
 
 				default:
 					console.log(`Unexpected event ${event.code}`);
-			}
+				}
```

Comment:
this doesn't seem to be properly aligned anymore

Reply:
true. eagle eyes!